### PR TITLE
BeAIM: new recipe

### PR DESCRIPTION
**Pull request by bach5000**
Needs license and copyright information, but can't find it, currently...

### CODE REVIEW

File: haiku-apps/beaim-1.5.6.recipe [hunk 1]
```diff
@@ -0,0 +1,42 @@
+SUMMARY="AIM chat client for BeOS"
+DESCRIPTION="BeAIM is an AOL Instand Messenger client that allows you \
+to chat, one-on-one, with other AIM or AOL users, and to keep tabs on when \
+they are online. Unlike its Windows or Mac alternatives, it has no ads!"
+COPYRIGHT=""
```
Comment (fbrosson) [inline haiku-apps/beaim-1.5.6.recipe]:
```diff
+HOMEPAGE="https://www.csh.rit.edu/~gile/projects/beaim/home.html"
 COPYRIGHT=""
```

File: haiku-apps/beaim-1.5.6.recipe [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+LICENSE=""
+REVISION="1"
+srcGitRev="6d98b0004d2980dffbd5b571d1390724a408b0c2"
+SOURCE_URI="https://github.com/HaikuArchives/beaim/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="3bb2629129be0eb8f7f06a8e37d027b3e46c4fabfe0cd0dd654c445ecc093c41"
+SOURCE_DIR="BeAIM-$srcGitRev"
```
Comment (fbrosson) [inline haiku-apps/beaim-1.5.6.recipe]:
```diff
 SOURCE_URI="https://github.com/HaikuArchives/beaim/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="3bb2629129be0eb8f7f06a8e37d027b3e46c4fabfe0cd0dd654c445ecc093c41"
+SOURCE_FILENAME="beaim-$portVersion-$srcGitRev.tar.gz"
 SOURCE_DIR="BeAIM-$srcGitRev"
```

Comment (bach5000) [reply haiku-apps/beaim-1.5.6.recipe]: Oops! I keep forgetting SOURCE_FILENAME.

**Comment by pulkomandy** [comment]
AIM servers were shut down last year. I see no need for adding this in haikuports.

**Comment by bach5000** [comment]
Guess I'll just close this PR.

**Pull request closed without merging**